manifests/init.pp

```diff
@@ -316,6 +319,24 @@
       mode    => '0444',
       content => $hints_file_content,
     }
+    if $update_root_hints {
```

Comment:
Should it be able to ensure the timer is absent?

Reply:
implemented it